lib/get-config.js

```diff
@@ -303,7 +305,25 @@ function validateRules(config, options) {
   }
 }
 
-module.exports = function(options) {
+/**
+ * Returns the rules config in conjunction with overrides configuration.
+ * @param {*} config
+ * @param {*} filePath
+ */
+function getRulesConfig(config, filePath) {
+  let configuredRules = config.rules;
+  let overrides = config.overrides;
+  if (filePath && overrides) {
+    let overridesObj =
+      overrides.find(override => override.files && micromatch.isMatch(filePath, override.files)) ||
+      {};
+    // If there is an override present, then the overridden ruleset takes precedence.
+    return Object.assign({}, configuredRules, overridesObj.rules);
+  }
+  return configuredRules;
```

Comment:
Let's return the full config object (remember we eventually want to support merging many things, not just rules)